docs/source/sdk/evernode/evdevkit/overview.md

```diff
@@ -178,6 +199,18 @@ evdevkit audit -h <host Xahau address>
 | EV_HP_OVERRIDE_CFG_PATH | (Optional) File path of the HotPocket configuration for the contract bundle upload.      |
 | EV_NETWORK              | (Optional) Evernode network to be used `(mainnet\|testnet)`. Default would be `mainnet`. |
 
+## Advanced usage
+```
+# Do [acquire](#acquiring-instance-from-evernode), [bundle](#creating-the-deployable-contract-package) and [deploy](#uploading-a-contract-to-evernode) in one command
```

Comment:
Let's mention that these params and options are same as acquire and deploy commands